ios/Capacitor/Capacitor/WebViewDelegationHandler.swift

```diff
@@ -62,9 +62,9 @@ internal class WebViewDelegationHandler: NSObject, WKNavigationDelegate, WKUIDel
 
     @available(iOS 15, *)
     func webView(_ webView: WKWebView,
-    requestDeviceOrientationAndMotionPermissionFor origin: WKSecurityOrigin,
-         initiatedByFrame frame: WKFrameInfo,
-          decisionHandler: @escaping (WKPermissionDecision) -> Void) {
+                 requestDeviceOrientationAndMotionPermissionFor origin: WKSecurityOrigin,
```

Comment:
looks like there is some problem with swiftlint or it's not happy with this lines anymore, but can you remove it from this PR?

Reply:
Yeah not sure what's up there. Removed it 👍